proteus/mprans/MCorr.py

```diff
@@ -102,6 +106,12 @@ def attachModels(self,modelList):
             self.ebq_H_vof = modelList[self.VOFModelIndex].ebq[('u',0)]
         else:
             self.ebq_H_vof = None
+
+        #: YY
```

Comment:
You don't need to mark your contributions with YY. Git tracks that for you. See `git blame`.